client/src/index.jsx

```diff
@@ -152,17 +205,45 @@ class App extends React.Component {
     });
   }
 
+  handleSelectChange(e){
+    let currentValues = Object.assign({}, this.state.currentValues);
+    if(e.target.value === "15-year fixed"){
```

Comment:
space between parens
```suggestion
    if (e.target.value === "15-year fixed") {
```